utils/contract-deployment/README.md

```diff
@@ -17,4 +17,21 @@ OR
 `go run utils/contract-deployment/contractDeploymentTools.go deriveContractAddress 0x38545c4b331D8BFb3bee94C62D77a6735b5eF8c0 1`
 
 ## Results
-The resulting raw transaction, `TeleporterMessenger` contract address, and universal deployer address are written to standard output, as well as to `UniversalTeleporterDeployerTransaction.txt`, `UniversalTeleporterMessengerContractAddress.txt`, and `UniversalTeleporterDeployerAddress.txt` respectively.
\ No newline at end of file
+The resulting raw transaction, `TeleporterMessenger` contract address, and universal deployer address are written to standard output, as well as to `UniversalTeleporterDeployerTransaction.txt`, `UniversalTeleporterMessengerContractAddress.txt`, and `UniversalTeleporterDeployerAddress.txt` respectively.
+
+## Deploy the contract
+Now that the keyless transaction is constructed, fund the deployer address. For example, using `cast`:
+
+```bash
+teleporter_deploy_tx=$(cat UniversalTeleporterDeployerTransaction.txt)
+cast send --private-key $my_private_key --value 50ether $teleporter_deploy_address --rpc-url $my_rpc_url
```

Comment:
The current keyless transaction uses 10 native tokens for gas in all cases, so I'd recommend updating this to `10ether` so it's not larger than necessary.